pkgs/development/tools/packer/default.nix

```diff
@@ -1,25 +1,26 @@
-{ lib, buildGoPackage, fetchFromGitHub, installShellFiles }:
-buildGoPackage rec {
+{ lib, buildGoModule, fetchFromGitHub, installShellFiles }:
+
+buildGoModule rec {
   pname = "packer";
   version = "1.7.1";
 
-  goPackagePath = "github.com/hashicorp/packer";
-
-  subPackages = [ "." ];
-
   src = fetchFromGitHub {
     owner = "hashicorp";
     repo = "packer";
     rev = "v${version}";
     sha256 = "sha256-PZwKvb43Xf8HaC148Xo076u3sP53nwC4fJ2X7HU0gDo=";
   };
 
+  vendorSha256 = null;
+
+  subPackages = [ "." ];
+
   buildFlagsArray = [ "-ldflags=-s -w" ];
 
   nativeBuildInputs = [ installShellFiles ];
 
   postInstall = ''
-    installShellCompletion --zsh go/src/${goPackagePath}/contrib/zsh-completion/_packer
+    installShellCompletion --zsh contrib/zsh-completion/_packer
```

Comment:
I think this is no longer installed correctly as I can't find it in results.

Reply:
It ends up in results/share correctly for me after `nix-build -A packer`ing?

Reply:
Looks fine to me:
```shell
$ ls -R result/
result/:
bin  share

result/bin:
packer

result/share:
zsh

result/share/zsh:
site-functions

result/share/zsh/site-functions:
_packer
```

Reply:
I think I was looking into cryptomator results.